src/components/SearchContainer/SearchContainer.spec.js

```diff
@@ -0,0 +1,41 @@
+import SearchContainer from './SearchContainer';
+import { MemoryRouter } from 'react-router';
+import Router from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: jest.fn(),
+  useLocation: jest.fn(),
+}));
+
+describe('Search testing', () => {
+  jest.spyOn(Router, 'useLocation').mockReturnValue({ search: '' });
+  test('renders Search component', () => {
+    const TEST_VALUE = 'test';
+    jest.spyOn(Router, 'useParams').mockReturnValue({ searchValue: TEST_VALUE });
+    render(<SearchContainer />);
+    // screen.debug();
```

Comment:
Nitpick: please remove the comment.

Reply:
I knew you will leave comment out on that, however if you don't mind I would leave it how it is. In production code I do not leave such comments in code but here I would like to have it. I left it intentionally, because I use screen for the first time, and I was impressed with debug feature, that's why I left it to remember to use it in future.